subprojects/docs/src/docs/userguide/declaringDependencies.adoc

```diff
@@ -43,6 +43,16 @@ A typical example for such a library in a Java project is the link:https://proje
 
 A Gradle project can define other types of repositories hosting binary dependencies. You can learn more about the syntax and API in the section on <<sec:declaring_repositories,declaring repositories>>. Refer to <<java_plugin>> for a deep dive on declaring dependencies for a Java project. The resolution behavior for binary dependencies declarations is <<sec:customizing_dependencies,highly customizable>>.
 
+==== Declaring a binary dependency without version
```

Comment:
Are we going to stick with "binary"?

Reply:
Should we make a reference to Maven's BOM support and how it is different or similar?

Reply:
I used the term "binary dependency" because it was used in all the other titles. I removed all usages in a separate commit: https://github.com/gradle/gradle/pull/4104/commits/56b88ec94512ed7068d395eff1bd68ec2f18abfb

What do you think?

Reply:
> Should we make a reference to Maven's BOM support and how it is different or similar?

Yes I think so. I opened a separate issue. If you have any more thoughts about it (e.g. where the content fits into the structure), please comment on that issue:  #4189